source/localizable/tutorial/autocomplete-component.md

```diff
@@ -102,6 +102,7 @@ Now, define your new controller like so:
 ```app/controllers/index.js
 export default Ember.Controller.extend({
   filteredList: null,
+  model: null,
```

Comment:
The error is related with [another problem](https://github.com/emberjs/guides/pull/1264), we're currently reviewing the whole guide.